ftplugin/purescript_pscide.vim

```diff
@@ -501,10 +501,21 @@ endfunction
 
 " Add type annotation
 function! PSCIDEaddTypeAnnotation(ident)
-  call s:getType(
-    \ a:ident,
-    \ v:true,
-    \ { resp -> s:PSCIDEaddTypeAnnotationCallback(a:ident, resp) }
+  let currentModule = purescript#ide#utils#currentModule()
```

Comment:
Why not just change `s:getType`.  This will also fix `Ptype` command.

Reply:
How? add to `s:getType` new parameter with meaning `use currentModule as only filter`?

Reply:
I think it would be just enough to add the `currentModule` to the `modules` list [here](https://github.com/FrigoEU/psc-ide-vim/blob/master/ftplugin/purescript_pscide.vim#L699)